app/routes/meetings/edit.js

```diff
@@ -6,8 +6,38 @@ export default class MeetingsEditRoute extends Route {
 
   async model(params) {
     const zitting = await this.store.findRecord('zitting', params.id, {
-      include: 'bestuursorgaan,secretaris,voorzitter,intermissions',
+      include:
+        'bestuursorgaan,secretaris,voorzitter,intermissions,publicatie-agendas',
```

Comment:
```suggestion
        'bestuursorgaan,secretaris,voorzitter,intermissions',
```
I think it's kind of odd to include publicatie-agendas here and not the other ones? either load all of them as includes or none.